Example/SmileID/RootView.swift

```diff
@@ -11,6 +12,14 @@ struct RootView: View {
     init() {
         UITabBar.appearance().barTintColor = UIColor(SmileID.theme.backgroundLight)
         UITabBar.appearance().tintColor = UIColor(SmileID.theme.accent)
+        if let dsn = Bundle.main.object(forInfoDictionaryKey: "SentryDSN") as? String {
+            SentrySDK.start { options in
```

Comment:
We should not put this here, this might get called multiple times due to recomposition (in practice it seems like most SwiftUI's initializers are called twice in direct succession)

According to Sentry docs, we should put this either in application didFinishLaunchingWithOptions, or in a SwiftUI `App` sublcass. We don't have the subclass at the moment though, so I think applicationDidFinishLaunchingWithOptions would be the better place